ui/app/components/app/network-display/network-display.js

```diff
@@ -78,6 +87,13 @@ NetworkDisplay.propTypes = {
   labelProps: PropTypes.shape({
     ...Chip.propTypes.labelProps,
```

Comment:
I think this proptype is messed up :thinking: 

I keep seeing this error in the console: 
```
Warning: Failed prop type: The prop `labelProps.isRequired` is marked as required in `NetworkDisplay`, but its value is `undefined`.
    in NetworkDisplay (created by ConfirmationPage)
    in ConfirmationPage (created by Context.Consumer)
    in Route (created by Authenticated)
    ...
```

Presumably introduced in a different PR so maybe out of scope, but I'm just mentioning because it kept catching my eye in testing.

Reply:
I figured it out, lol. that particular prop (labelProps) is already a shape, it didn't need to be spread into a new shape. that caused....irregularities. i'm surprised it even worked without a more interesting error!